### ISSUE TITLE
create release pipeline on main branch

### ISSUE THREAD

**Issue by seebi** [eccenca/jod#12]
**Is your feature request related to a problem? Please describe.**

We need a release pipeline in order to release jod automatically with github mechanics.

**Comment by white-gecko** [comment]
There is a pipeline to build a docker/podman image available now, the gem release is still missing.

**Comment by white-gecko** [comment]
No, gem should be published as well and should not be to complicated, cf. https://github.com/AKSW/jekyll-rdf/blob/develop/.travis.yml, this did publish gems back then.

**Comment by white-gecko** [comment]
Maybe https://github.com/marketplace/actions/publish-to-rubygems
